sky/resources.py

```diff
@@ -275,47 +275,16 @@ def _set_region_zone(self, region: Optional[str],
         self._region, self._zone = self._cloud.validate_region_zone(
             region, zone)
 
-    def get_valid_region_zones_for_launchable(
-            self) -> List[Tuple[clouds.Region, List[clouds.Zone]]]:
+    def get_valid_region_zones_for_launchable(self) -> Set[clouds.Region]:
```

Comment:
Should we now name it `get_valid_regions_for_launchable()` or `get_offering_regions_for_launchable()`?

Should we remove "zone" on L279?

Reply:
Fixed.